README.md

```diff
@@ -30,6 +30,7 @@ The documentation is divided into several sections:
 * [Main Concepts](https://reactjs.org/docs/hello-world.html)
 * [Advanced Guides](https://reactjs.org/docs/jsx-in-depth.html)
 * [API Reference](https://reactjs.org/docs/react-api.html)
+* [Hooks(Preview)](https://reactjs.org/docs/hooks-intro.html)
```

Comment:
👋 Should there be a space before the opening bracket?

Reply:
If the hooks are officially released in the future, there will be no problem since it will disappear.

Reply:
```suggestion
* [Hooks (Preview)](https://reactjs.org/docs/hooks-intro.html)
```